lib/grinder_tools.dart

```diff
@@ -139,10 +139,10 @@ Future runProcessAsync(GrinderContext context, String executable,
  * A default implementation of an `init` task. This task verifies that the grind
  * script is executed from the project root.
  */
-void defaultInit(GrinderContext context) {
+void defaultInit([GrinderContext context]) {
```

Comment:
For the APIs that take a GrinderContext parameter which is now optional, can we mark it as deprecated?